readthedocs/projects/filters.py

```diff
@@ -0,0 +1,158 @@
+import logging
+
+from django.db.models import Count, F, Max
+from django.forms.widgets import HiddenInput
+from django.utils.translation import ugettext_lazy as _
+from django_filters import CharFilter, ChoiceFilter, FilterSet, OrderingFilter
+
+log = logging.getLogger(__name__)
+
+
+class SortOrderingFilter(OrderingFilter):
+
+    """
+    Version list sort ordering django_filters filter.
+
+    Django-filter is highly opionated, and the default model filters do not work
+    well with empty/null values in the filter choices. In our case, empty/null
+    values are used for a default query. So, to make this work, we will use a
+    custom filter, instead of an automated model filter.
```

Comment:
By reading this comment, I'm not sure to understand what's the problem that this filter solves 😅 --I only get that it does "something" with null values, but not sure what fixes from the default filter.

Reply:
:+1: I can expand on this